tests/test_data/test_datasets/test_num_class_hook.py

```diff
@@ -0,0 +1,83 @@
+import logging
```

Comment:
It's better to place this file in the `tests/test_runtime` folder since we have placed some unit tests of hooks there.